gm4_lively_lily_pads/data/gm4_lively_lily_pads/guidebook/lively_lily_pads.json

```
@@ -0,0 +1,61 @@
+{
+  "id": "lively_lily_pads",
+  "name": "Lively Lily Pads",
+  "module_type": "module",
+  "icon": {
+    "id": "minecraft:lily_pad"
+  },
+  "criteria": {
+    "obtain_pad": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "minecraft:lily_pad"
+            ]
+          }
+        ]
+      }
+    },
+    "obtain_supported_item": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": "#gm4_lively_lily_pads:supported_item"
+          }
+        ]
+      }
+    }
+  },
+  "sections": [
+    {
+      "name": "display",
+      "enable": [],
+      "requirements": [
+        [
+          "obtain_pad",
+          "obtain_supported_item"
+        ]
+      ],
+      "pages": [
+        [
+          {
+            "insert": "title"
+          },
+          {
+            "translate": "text.gm4.guidebook.lively_lily_pads.description",
+            "fallback": "While holding a supported item and looking at a lily pad, simply right click on the green particles to place it down.\nCandles will need lighting."
```

Try something more player oriented, e.g.

```suggestion
            "fallback": "Simply right click lily pads with decorations to add them on top! \nCandles will need lighting."
```

---

I've used this but changed it slightly to
`Simply right click the particles on lily pads with decorations to add them on top!\nCandles will need lighting.`
I feel mentioning the particles is important as they don't appear right away.